apps/central-scan/backend/src/store.ts

```diff
@@ -599,13 +599,10 @@ export class Store {
     [front, back]: SheetOf<PageInterpretationWithFiles>
   ): string {
     try {
-      const finishedAdjudicationAt =
-        front.interpretation.type === 'InterpretedHmpbPage' &&
-        back.interpretation.type === 'InterpretedHmpbPage' &&
-        !front.interpretation.adjudicationInfo.requiresAdjudication &&
-        !back.interpretation.adjudicationInfo.requiresAdjudication
-          ? DateTime.now().toISOTime()
-          : undefined;
```

Comment:
Because the finished date was using different logic than the flag, it could lead to situations in which the ballot was immediately marked as needing adjudication and adjudicated.